### PR TITLE
healthbot: stop using on validator transaction relaying

### PR DESCRIPTION
**Pull request by jsign**
# Summary

Closes https://github.com/tablelandnetwork/go-tableland/issues/400

# Context

In the upcoming API v1, validators won't be offering transaction relaying. Our current Healthbot implementation 

**Note: this PR is ready for review, but keeping it as draft until Bruno comes back**

# Implementation overview

The implementation goes along the lines of a previously stalled attempt #279 with some slight changes.

The infrastructure work we have to in each validator is to update the `deployed/<env>/healthbot/.env_healthbot` files removing the previous `HEALTHBOT_<chainName>_SIWE` and adding `HEALTHBOT_<chainName>_PRIVATE_KEY`.

# Implementation details and review orientation

NA

# Checklist
- [X]  Are changes backward compatible with existing SDKs, or is there a plan to manage it correctly?
- [X]  Are changes covered by existing tests, or were new tests included?
- [X]  Are code changes optimized for future code readers, commenting on problematic areas to understand (if any)?
- [X]  Future-self question: Did you avoid unjustified/unnecessary complexity to achieve the goal?

### CODE REVIEW

File: pkg/client/chains.go [hunk 1]
```diff
@@ -40,64 +40,64 @@ var ChainIDs = struct {
 type Chain struct {
```
[the remark this answers is not in the document]
Comment (asutula) [reply pkg/client/chains.go:40]: Ok sounds good. There's something I like about the primary entry point to referencing chains being a human readable name (`EthereumGoerli`, for example), but I'm fine with trying this out.

[the remark this answers is not in the document]
Comment (asutula) [reply pkg/client/chains.go:40]: I realize mapping from strings in a config file to a struct member name is harder, more error prone, and more tedious. Using chain ids is definitely easier in that way.

**Review by asutula — APPROVED** [review]
Good work! Looks familiar, I like it ;)

File: docker/deployed/testnet/healthbot/config.json [hunk 2]
```diff
@@ -6,41 +6,41 @@
         "Human": false,
         "Debug": true
     },
-    "Target": "https://testnet.tableland.network/rpc",
+    "Target": "https://testnets.tableland.network/rpc",
```
Comment (brunocalza) [inline docker/deployed/testnet/healthbot/config.json:9]: Should we remove the`Target` entry in this config? (also in `staging`)

**Review by brunocalza — APPROVED** [review]
Nice! The code became much simpler with the client :)

File: docker/deployed/testnet/healthbot/config.json (hunk 2 above)
Comment (jsign) [reply docker/deployed/testnet/healthbot/config.json:9]: Good point, will do.

Comment (jsign) [reply docker/deployed/testnet/healthbot/config.json]: @brunocalza, done, added an extra commit. Ready to have another approval for merging.

**Review by brunocalza — APPROVED** [review]

**Pull request merged**